packages/ibm-products/src/components/CreateTearsheetNarrow/CreateTearsheetNarrow.stories.js

```diff
@@ -299,6 +299,5 @@ export const withMultipleForms = prepareStory(WithValidationTemplate, {
   storyName: withMultipleFormsStoryName,
   args: {
     ...defaultStoryProps,
-    multiForm: true,
```

Comment:
`multiForm` does not appear as a prop anywhere in Carbon for IBM Products.

(There is a `MultiFormEditTearsheet` component. In all likelihood, this component replaced the prop and this `multiForm` prop was missed during the clean up.